webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -4100,6 +4101,116 @@ const firstVisibleMsg = {
+                        return totalMinutes;
+                    } catch (e) {
+                        // if there is any error in parsing the timezone offset return GMT offset
+                        console.debug('Timezone not avaiable in team settings');
+                        return 0;
+                    }
+                };
+
+                const now = new Date();
+                const currentDay = now.getDay();
+                const offset = now.getTimezoneOffset();
+                const gmtTime = new Date(now.getTime() + offset * 60000);
+                const timezoneOffset = parseTimezoneOffset(team.timezone);
+                const adjustedTime = new Date(
+                    gmtTime.getTime() + timezoneOffset * 60000
+                );
+                const businessHours = team.businessHourMap[currentDay];
+                if (!businessHours) {
+                    // No business hours for the current day
+                    return false;
+                }
+                try {
+                    const [start, end] = businessHours
+                        .split('-')
+                        .map((time) => convertToMinutes(time));
+                    const currentTimeInMinutes =
+                        adjustedTime.getHours() * 60 + adjustedTime.getMinutes();
+                    // Check if the current time is within the business hours range
+                    if (start <= end) {
+                        return (
+                            currentTimeInMinutes >= start &&
+                            currentTimeInMinutes <= end
+                        );
+                    } else {
+                        // Handle case where the business hours wrap around midnight (e.g., 2200-0400)
+                        return (
+                            currentTimeInMinutes >= start ||
+                            currentTimeInMinutes <= end
+                        );
+                    }
+                } catch(e) {
+                    // if there is any error in formatting the business hours allow the user to chat
+                    console.error('Error while checking business hours', e);
+                    return true;
+                }
+            };
+
+            _this.handleBusinessHours = function () {
+                var currentGroupData = CURRENT_GROUP_DATA || {};
+                window.Applozic.ALApiService.ajax({
+                    type: 'GET',
+                    url: MCK_BASE_URL + BUSINESS_HOURS_URL,
+                    global: false,
+                    contentType: 'application/json',
+                    success: function (data) {
+                        let teamSettings = {};
+                        const response = data?.response;
+                        if (!currentGroupData.teamId) {
+                            teamSettings = response.find(
+                                (team) => team.teamName === 'Default Team'
```

Comment:
move this to constant

Reply:
added constant to check this, also added now checking both default and default team